settings.gradle

```diff
@@ -127,9 +127,13 @@ project(":code/priority-queue").name = "priority-queue"
 // Тамилина Дарья
 // Тягунов Владимир
 // Федорова Дарья
-// Федотов Андрей
```

Comment:
Revert changes, pls

Reply:
You should add new dependencies here

Reply:
It was said to change comment to project name